solidity/contracts/hooks/igp/InterchainGasPaymaster.sol

```diff
@@ -218,6 +254,22 @@ contract InterchainGasPaymaster is
         return _gasOracle.getExchangeRateAndGasPrice(_destinationDomain);
     }
 
+    /**
+     * @notice Returns the stored destinationGasOverhead added to the _gasAmount.
+     * @dev If there is no stored destinationGasOverhead, 0 is used.
+     * @param _destinationDomain The domain of the message's destination chain.
+     * @param _gasAmount The amount of destination gas to pay for. This should not
+     * consider any gas that is accounted for in the stored destinationGasOverhead.
+     * @return The stored destinationGasOverhead added to the _gasAmount.
```

Comment:
"should not consider any gas that is accounted for in the stored overhead" is not very clear
can you make the distinction more obvious? eg system overhead vs application overhead (ISM verification or message handling)

Reply:
this is @tkporter's comment.  The contract @dev comment clarifies it. I've added the "application-sepcific" part.